ui/src/components/yaml/index.tsx

```diff
@@ -254,6 +255,8 @@
       setStreaming(false)
```

Comment:
Consider adding a null check before accessing `getBoundingClientRect()` to handle cases where the ref might be null.